app/scrapers/buildings.rb

```diff
@@ -1,5 +1,6 @@
-# get umd building gis data from zfogg's github gist. Not really the ideal long-term solution...
-# https://gist.githubusercontent.com/zfogg/4bc03d7f71d5f740d028/raw/afe9f0baeda4ef6a7a64d99fa14bded8eb6bf3a8/umd-building-gis.json
+# This pulls GIS data from this GitHub Repo, originally from zfogg
+# https://raw.githubusercontent.com/umdio/umdio/master/app/data/umd-building-gis.json
+# TODO: Find a UMD source to pull this data from, rather than manual updates
```

Comment:
If the data is committed, then it might make sense to deprecate this scraper and load the data as json from the filesystem instead of having to go over the network - that was a bad hack when I first wrote it, though I suppose mostly costless

Reply:
It might actually be a better approach for me to not commit the file, and instead just host the updated one on a gist for some time. It would also avoid cluttering up the repo, because long-term we can hopefully actually scrape data and get a lot more useful information, rather than having to add it all by hand